--- a/meta/recipes-devtools/pseudo/pseudo_git.bb
+++ b/meta/recipes-devtools/pseudo/pseudo_git.bb
@@ -12,9 +12,9 @@
     file://older-glibc-symbols.patch"
 SRC_URI[prebuilt.sha256sum] = "ed9f456856e9d86359f169f46a70ad7be4190d6040282b84c8d97b99072485aa"
 
-SRCREV = "56e1f8df4761da60e41812fc32b1de797d1765e9"
+SRCREV = "4194895cd61b439e47e88820c3229767037cdc69"
 S = "${WORKDIR}/git"
-PV = "1.9.3+git"
+PV = "1.9.9"
 
 # largefile and 64bit time_t support adds these macros via compiler flags globally
 # remove them for pseudo since pseudo intercepts some of the functions which will be
